api/src/models/index.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
```

Comment:
só pra entender mesmo, você forçou strict mode em algumas arquivos e em outros não, foi de propósito?